Sources/NostrSDK/Events/RecommendServerEvent.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  RecommendServerEvent.swift
+//  
+//
+//  Created by Bryan Montz on 7/23/23.
+//
+
+import Foundation
+
+/// An event that contains a relay the event creator wants to recommend to its followers.
+///
+/// > Note: [NIP-01 Specification](https://github.com/nostr-protocol/nips/blob/b503f8a92b22be3037b8115fe3e644865a4fa155/01.md#basic-event-kinds)
+final class RecommendServerEvent: NostrEvent {
+    var relayURL: URL? {
```

Comment:
Would there be any validation we want to perform here to ensure it's a websocket URL (i.e., `ws://` or `wss://`) and return `nil` if not?

Reply:
Not sure. We could if we want because the spec specifically says it's for relays and those will be websockets for now.

Reply:
For this specific case, I think we can be opinionated here because any other non-websocket protocol URL does not make on this field.